R/html-tweak.R

```diff
@@ -172,11 +175,35 @@ tweak_homepage_html <- function(html, strip_header = FALSE, sidebar = TRUE) {
   if (strip_header) {
     xml2::xml_remove(header, free = TRUE)
   } else {
-    page_header_text <- paste0("<div class='page-header'>", header, "</div>")
+     page_header_text <- if (bs_version == 3) {
+      paste0("<div class='page-header'>", header, "</div>")
+    } else {
+      sprintf("<div class='pb-2 mt-4 mb-2 border-bottom'>%s</div>", header)
+      }
     page_header <- xml2::read_html(page_header_text) %>% xml2::xml_find_first("//div")
     xml2::xml_replace(header, page_header)
   }
 
+  # Logo tweaks
```

Comment:
this is new and meant to help make the logo more prominent

Reply:
Can we pull this out of this PR for now? I think it will need more discussion.

Reply:
#1561